src/command_modules/azure-cli-keyvault/azure/cli/command_modules/keyvault/_help.py

```diff
@@ -54,6 +54,57 @@
     short-summary: Manage certificates.
 """
 
+helps['keyvault certificate get-default-policy'] = """
+    type: command
+    short-summary: Get a default policy for a self-signed certificate
+    long-summary: >
+        This default policy can be used in conjunction with `az keyvault create` to create a self-signed certificate.
+        The policy can also be used as a base to manipulate to create other permutations of Key Vault certificates.
+    examples:
+        - name: Create a self-signed certificate with a the default policy
+          text: >
+            az keyvault create -g group-name -n vaultname -l westus --enabled-for-deployment true \\
+              --enabled-for-template-deployment true
+
+            az keyvault certificate create --vault-name vaultname -n cert1 \\
+              -p "$(az keyvault certificate get-default-policy)"
+"""
+
+helps['keyvault certificate create'] = """
+    type: command
+    long-summary: >
+        Create a Key Vault certificate. Certificates can also be used as a secrets in provisioned virtual machines.
+    examples:
+        - name: Create a self-signed certificate with a the default policy and add to a virtual machine
+          text: >
+            az keyvault create -g group-name -n vaultname -l westus --enabled-for-deployment true \\
+              --enabled-for-template-deployment true
+
+            az keyvault certificate create --vault-name vaultname -n cert1 \\
+              -p "$(az keyvault certificate get-default-policy)"
+
+            az vm create -g group-name -n vm-name --admin-username deploy --image debian --secrets \'{secrets}\''
+"""
+
+helps['keyvault secret vm-format'] = """
```

Comment:
Based on our discussion yesterday, this is definitely a VM (Windows VM) specific thing. However, I think we should strongly consider having this command be something like: `az vm/vmss get-keyvault-reference` or something similar. While the command logic would remain in the KeyVault module, the "natural" place for these types of commands to live in the command tree is in the module they pertain to (vm, appservice, sql, etc). These commands would only appear in these branches if the KeyVault module were installed (which, by default, it is).